src/player.py

```diff
@@ -1,2 +1,8 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
```

Comment:
You might consider refactoring to conform better to OOP design... for instance, if a player can move, see, or perform any action, that function might better live on the player such that you can get player.moves_right or player.pickups_up_object as opposed to that logic or action living in a different object that might contain the player object, but not make choices for the player... if that makes sense. When a code base explodes (um, unlike this project haha) conforming to such a pattern makes it reeeeaaaalllly easy to know where to look to find and fix things. As I understand it, this is where the entire idea of orienting around the object comes from. In which case, you might consider moving a lot fo f the logic in adv.py to player.py